### ISSUE TITLE
Add "Enable clickthrough" setting for hotbars

### ISSUE THREAD

**Issue by Murkchan** [Caraxi/SimpleTweaksPlugin#80]
I immediately apologize for the text from the translator

Add "Enable clickthrough" setting for hotbars
Moreover, you need settings for a certain hotbar.

Since there are no normally functioning addons for tracking cooldowns, easier to put /micon macros to the hotbar.
However, since the hotbar is active, misсlicks may occur.

**Comment by gamer4eg** [comment]
I agree with the author of the issue. The basic interface of the Ff14 lacks a handy ability to track ability cooldowns. The method with a separate strip for this is not inconvenient because of the impossibility of clicking through it, and also because it reacts to mouse hover.
